dsl/src/commonTest/kotlin/it/unibo/collektive/aggregate/NeighboringTest.kt

```diff
@@ -29,6 +29,13 @@ class NeighboringTest : StringSpec({
         }
     }
 
+    "Neighboring without messages with efficient neighboring implementation" {
```

Comment:
It does not really communicate any intent...

Reply:
You are right... I have prepended the string to differentiate the test case but it makes no sense.
I'll change all the test cases in this file to be more explicit.